lib/helpers.coffee

```diff
@@ -61,7 +61,7 @@ module.exports = Helpers =
       regex = xcache.get(rawRegex)
     else
       xcache.set(rawRegex, regex = XRegExp(rawRegex))
-    for line in data
+    for line in data.split(/\r?\n/g)
```

Comment:
I don't believe the regular expression needs the `g` flag.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/split#Removing_spaces_from_a_string